test/apple_bundle_version_test.sh

```diff
@@ -48,8 +48,8 @@ saver = rule(
 EOF
 
   cat > pkg/BUILD <<EOF
-load("@build_bazel_rules_apple//apple:versioning.bzl",
-     "apple_bundle_version")
+load("@build_bazel_rules_apple//apple:providers.bzl",
```

Comment:
this is actually ok, since it's loading the rule, not the provider

Reply:
Doh, will update soon (just walked away from my laptop)

Reply:
Updated now!